mhkit/wave/resource.py

```diff
@@ -36,15 +36,22 @@ def elevation_spectrum(eta, sample_rate, nnft, window='hann',
 
     # TODO: Add confidence intervals, equal energy frequency spacing, and NDBC
     #       frequency spacing
-    # TODO: may need an assert for the length of nnft- signal.welch breaks when nfft is too short
+    # TODO: may need to raise an error for the length of nnft- signal.welch breaks when nfft is too short
     # TODO: check for uniform sampling
```

Comment:
Unrelated to this PR and mostly a note to self but we shouldn't have TODOs in production code. These should be captured in issues and removed.